.eslintrc

```diff
@@ -20,6 +20,8 @@
     "sum": true,
     "count": true,
     "gapi": true,
+    "Set": true,
+    "ga": true,
```

Comment:
Next time we update permissions, we're going to switch to the new permissions to use the latest 'ga'.